git/gitbitbucketclient.py

```diff
@@ -27,9 +34,15 @@ def _commitData(self, commit):
 		:param commit: commit object
 		:type  commit: git.objects.commit.Commit
 		"""
-		pass
+		return {
+			"hexsha": commit['hash'],
+			"adate": commit['date'],
+			"cdate": commit['date'],
+			"author": commit['author']['raw'],
+			"message": commit['message']
+		}
 
-	def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.timedelta(hours=24)):
+	def commits(self, branch, since = 0, to = time.mktime((datetime.datetime.now() + datetime.timedelta(hours=24)).timetuple())):
```

Comment:
Yeah, you are right. I overlooked the mktime is around timetuple as well. Too many parenthesis :).